pom.xml

```diff
@@ -66,19 +66,21 @@
       explicitly specifying the latest version here because one we get from the parent POM
       tends to lag behind a bit
     -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
```

Comment:
Usually we try to keep 2y or 1y core, assuming that many plugins were updated for workflow plugin core requirements. Keeping 2 or 3 last lts versions should ok. I don't know any bugs that blocks people not to use 1.565.3, so imho 1.565.3 is ok.